templates/documents/file_upload_modal.html

```diff
@@ -0,0 +1,77 @@
+<dialog id="chooseFilesModal" class="modal" x-data="modalData">
+    <div class="modal-box max-w-none w-1/2 max-h-3/5 overflow-y-auto">
+        <h3 class="font-bold text-lg mb-4">Add Files to Collection</h3>
+        <form method="dialog">
+            <button class="btn btn-sm btn-circle btn-ghost absolute right-2 top-2">✕</button>
+        </form>
+        <form class="mt-2 flex flex-col gap-5" action="{% url 'documents:add_collection_files' team.slug collection.id %}" method="POST" enctype="multipart/form-data">
+            {% csrf_token %}
+            {% if collection.is_index %}
+                <fieldset class="fieldset mt-4">
+                    <legend class="fieldset-legend">Chunking Strategy</legend>
+                    <input name="chunk_size" type="text" class="input" value="800" required />
+                    <p class="label">Chunk Size (tokens)</p>
+                    <input name="chunk_overlap" type="text" class="input" value="800" required />
```

Comment:
This value is changed to 400 in [this PR](https://github.com/dimagi/open-chat-studio/pull/1560). It doesn't make sense for this value to be the same as the chunk size